.github/workflows/test.yml

```diff
@@ -5,35 +5,41 @@ name: test
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    name: Run tests (${{ matrix.os }}, pytorch version ${{ matrix.pytorch-version }}, python version ${{ matrix.python-version }})
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
         pytorch-version: ["master", "stable"]
```

Comment:
I think we don't need to test two versions. I never use the "master" version.

Reply:
Done.